server/lib/search_by_position.js

```diff
@@ -0,0 +1,43 @@
+const CONFIG = require('config')
+const __ = CONFIG.universalPath
+const _ = __.require('builders', 'utils')
+const { buildSearcher } = __.require('lib', 'elasticsearch')
+const assert_ = __.require('utils', 'assert_types')
+
+module.exports = (db, dbBaseName) => {
+  const searchByPosition = buildSearcher({
+    dbBaseName,
+    queryBuilder
+  })
+
+  return async bbox => {
+    assert_.numbers(bbox)
+    const hits = await searchByPosition(bbox)
+    const ids = _.map(hits, '_id')
+    return db.byIds(ids)
+  }
+}
+
+// See https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-geo-bounding-box-query.html
+const queryBuilder = ([ minLng, minLat, maxLng, maxLat ]) => {
+  const topLeft = { lat: maxLat, lon: minLng }
+  const bottomRight = { lat: minLat, lon: maxLng }
+  return {
+    query: {
+      bool: {
+        must: {
+          match_all: {}
+        },
```

Comment:
is an empty `match_all` necessary ?

Reply:
that's what the documentation proposes https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-geo-bounding-box-query.html
but it works when removing it (2bb13766 and e2215d8d), as long as we don't set a `minScore`, as the `filter` doesn't give any matching score (`hit._score = 0`)